android-sdk/android/src/main/kotlin/io/logto/sdk/android/LogtoClient.kt

```diff
@@ -69,26 +74,43 @@ open class LogtoClient(
             logtoConfig = logtoConfig,
             oidcConfig = requireNotNull(oidcConfig),
```

Comment:
nit: would it be cleaner to check the not null oidcConfig ahead of the SignInSession call instead of inline assertation?

Reply:
Our callback logic ensures that this `oidcConfig` is not null, but the type of `oidcConfig` is nullable.
So I think it's redundant to check oidcConfig is not null ahead. Just crash if `oidcConfig` is `null` here, for there is no point in handling this exception.